pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -25,14 +25,31 @@ in
 
 generated:
 
-with generated;
-
-{
+rec {
+
+  # list of plugins that are needed as dependencies somewhere
+  # This is necessary (as opposed to `with generated;`) because
+  # there would otherwise be conflicts with `rec`. `rec` is necessary
+  # however to make sure overriden plugins can be used as dependencies.
+  # This should use a proper package set with fixed-points instead.
```

Comment:
@FRidh would it be relatively simple to do that here? Basically `generated.nix` should generate a package set and this should be an overlay to that package set.